From 827297f1cc8b34d928580ccaa017860e24cfb9d6 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sun, 14 Oct 2018 18:41:30 +0300
Subject: [PATCH] Linux: I2C bus update number

We enabled I2C master0.
Now Logic bus number of power1220 device is bus1
---

diff --git a/board/habanalabs/common/rootfs_overlay/etc/init.d/S99misc b/board/habanalabs/common/rootfs_overlay/etc/init.d/S99misc
index 3641d87..9d47028 100755
--- a/board/habanalabs/common/rootfs_overlay/etc/init.d/S99misc
+++ b/board/habanalabs/common/rootfs_overlay/etc/init.d/S99misc
@@ -6,7 +6,7 @@
 case "$1" in
   start)
 	echo "Starting misc..."
-	echo powr1220 0x25 > /sys/bus/i2c/devices/i2c-0/new_device
+	echo powr1220 0x25 > /sys/bus/i2c/devices/i2c-1/new_device
 	#echo ir35221 0x40 > /sys/bus/i2c/devices/i2c-0/new_device
 	echo 0 > /sys/module/rcupdate/parameters/rcu_cpu_stall_suppress
 	for tz in /sys/class/thermal/thermal_zone*;
